bin/SConsDoc.py

```diff
@@ -141,22 +141,12 @@
         pass
 if not has_etree:
     try:
-        # Python 2.5
```

Comment:
I think you can remove the code as well as that's the python2 version of it?

Reply:
I'm holding out at the moment as the most reliable way to build `doc/generated` is using Python 2... that still works, but wouldn't if I dropped that bit.

Reply:
If you can put these back in #python.. then I'll merge. Better to leave the notes that the code is for py 27 until the code is removed as well.